PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs

```diff
@@ -6,6 +6,51 @@ namespace PrincessBrideTrivia.Tests;
 public class ProgramTests
 {
     [TestMethod]
+public void IncorrectQuestions_AreRecordedWhenAnsweredWrong()
+{
+    // Arrange
+    Question question1 = new()
+    {
+        Text = "What is my name?",
+        Answers = new string[] { "Chris", "Bill", "Frank" },
+        CorrectAnswerIndex = "1"
+    };
+
+    List<Question> incorrectQuestions = new List<Question>();
```

Comment:
```suggestion
    List<Question> incorrectQuestions = new();
```
nit: initialization can be simplified